.rubocop.yml

```diff
@@ -1,39 +1,28 @@
-require:
-  - rubocop-minitest
-  - rubocop-performance
-  - rubocop-rails
-
 inherit_from: .rubocop_todo.yml
 
+inherit_gem:
+  theforeman-rubocop:
+    - rules/base.yml
+    - rules/ruby_target.yml
+    - rules/style.yml
+    - rules/metrics.yml
+    - rules/performance.yml
+    - rules/rails.yml
+    - rules/minitest.yml
+    - rules/style_lenient.yml
+
 AllCops:
+  NewCops: disable
   Exclude:
-    - 'node_modules/**/*'
-    - 'locale/*'
-    - 'vendor/**/*'
```

Comment:
I decided to keep it as it was previously because of this failure:
```
app/models/foreman_tasks/concerns/action_triggering.rb:30:17: E: Lint/Syntax: unexpected token tDOT3
(Using Ruby 2.5 parser; configure using TargetRubyVersion parameter, under AllCops)
      def save!(...)
```
and added few cops offense count in the rubocop todo file.